addons/survey/static/src/js/survey_form.js

```diff
@@ -500,7 +525,10 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend({
                 // prevent enter submit if we're on a page description (there is nothing to submit)
                 this.preventEnterSubmit = true;
             }
-
+            // Background management - reset background overlay opacity to 0.7 to discover next background.
+            if (refreshBackground) {
+                $('div#wrapwrap').css("box-shadow", "inset 0 0 0 10000px rgba(255,255,255,0.7)");
```

Comment:
And here you would add "o_survey_background_transition o_survey_background_transition_light" ?
And the "_light" rule would only change the rgba in the scss file?

Beware that if you use this approach, you will have to add a couple removeClass as well to handle the back and forth.